wagtail/admin/templates/wagtailadmin/404.html

```diff
@@ -14,7 +14,9 @@
     <main class="page404__bg">
         <div class="w-w-full w-h-full w-max-w-6xl w-mx-auto w-flex w-items-center w-justify-center">
             {% block branding_logo %}
-                {% include "wagtailadmin/shared/animated_logo.html" %}
+                <div class="page404__logo w-hidden sm:w-block">
+                    <img src="{% versioned_static 'wagtailadmin/images/wagtail-logo.svg' %}" alt=""/>
+                </div>
```

Comment:
This `animated_logo.html` is no longer animated and is only used here.

Reply:
Maybe we should check any docs that reference overriding this template. Or add to upgrade considerations?

Reply:
The documented overrides are based on the `branding_logo` block so we should be good.